src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx

```diff
@@ -806,8 +806,47 @@ The .NET agent does not directly monitor datastore processes. Also, by default t
 >
   You can automatically instrument the following .NET languages and technologies:
   
-  * ADO.NET
-  * C# applications
+      <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            Language/technology
+          </th>
+
+          <th>
+            Additional configuration
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            C# applications
+          </td>
+          <td>
+            N/A
+          </td>
+        </tr>
+        <tr>
+          <td>
+            ADO.net
+          </td>
+          <td>
+            After installing the .NET agent, you need to [install the `nri-perfmon` integration](https://github.com/newrelic/nri-perfmon#installation) from the New Relic Open Source repo. 
+            
+            When you're ready, you have the option to forward your ADO.net logs by running the below command in the command line:
+
+            ```
+            NEW_RELIC_DISTRIBUTED_TRACING_ENABLED=true
+            NEW_RELIC_LICENSE_KEY=<var>YOUR_LICENSE_KEY</var>
+            NEW_RELIC_APP_NAME="ADO.NET" /usr/local/newrelic-netcore20-agent/run.sh dotnet run
```

Comment:
Here "ADO.NET" is an application name which we enter while going through the installation process so i think it may vary.

Reply:
Meaning that it should instead say `NEW_RELIC_APP_NAME="<VAR>YOUR_APP</VAR>"` rather than `"NEW_RELIC_APP_NAME="ADO.NET"`?

Reply:
Correct..

Reply:
Hi @akristen , so I have prepared installation process in steps, can you please have a look below and modify above steps:
Step 1. Install .NET agent
a. Go to [one.newrelic.com](http://one.newrelic.com/)
b. Click on “Add Data”
c. Search .Net Agent
d. Select an account and click on “Begin installation”
e. Click on any one option and follow further steps (Steps will depend on the option you choose)
Step 2: Install ‘nri-perfmon’ integration using [url](https://github.com/newrelic/nri-perfmon#installation) from the New Relic Open Source repo.
a. Follow the documentation mentioned in above url.
b. Run .exe file to track the performance
Step 3: Now you will able to see the logs on [newrelic](http://one.newrelic.com/) .

Reply:
@dikshab0202 and @pkosanam Currently talking over with @rhetoric101 whether these steps need to be represented. Will give an update here when we come to a conclusion